configs/cargo-deny/clarify.toml

```diff
@@ -28,27 +37,33 @@ skip-files = [
 ]
 
 [clarify.crossbeam-channel]
-expression = "(MIT OR Apache-2.0) AND BSD-2-Clause AND CC-BY-3.0"
+expression = "(MIT OR Apache-2.0) AND BSD-2-Clause-FreeBSD"
```

Comment:
Oh, in this case I took it from [clarify.toml](https://github.com/bottlerocket-os/bottlerocket/blob/develop/sources/clarify.toml#L48) in the core repo. It looks like it hasn't been accurate since https://github.com/crossbeam-rs/crossbeam/commit/33e4c64a5 removed the other license statement from `LICENSE-THIRD-PARTY`.

Reply:
Opened https://github.com/bottlerocket-os/bottlerocket/pull/2447 and https://github.com/bottlerocket-os/bottlerocket-update-operator/pull/250 to fix this elsewhere in the org.